src/swagger/http/handlers/web.cr

```diff
@@ -35,14 +35,14 @@ module Swagger::HTTP
 
     private def asset_uri?(context)
       original_path = context.request.path.not_nil!
-      is_dir_path = original_path.ends_with?("/")
-      request_path = self.request_path(URI.decode(original_path))
+      is_dir_path = original_path.ends_with? '/'
```

Comment:
**DO NOT** remote the brackets.

Reply:
Sorry I don't understand?

Reply:
Replacing a String by a Char you meant?
Yes sorry, no related to this PR - dropping this